app/Filament/Resources/HubResource.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Resources;
+
+use App\Filament\Components\BreadcrumbPreview;
+use App\Filament\Extensions\Resources\Resource;
+use App\Filament\Resources\HubResource\Pages;
+use App\Filament\Resources\HubResource\RelationManagers\GamesRelationManager;
+use App\Filament\Resources\HubResource\RelationManagers\ParentHubsRelationManager;
+use App\Models\GameSet;
+use App\Platform\Enums\GameSetType;
+use App\Support\Rules\NoEmoji;
+use Filament\Forms;
+use Filament\Forms\Form;
+use Filament\Infolists;
+use Filament\Infolists\Infolist;
+use Filament\Pages\Page;
+use Filament\Tables;
+use Filament\Tables\Table;
+use Illuminate\Database\Eloquent\Builder;
+
+class HubResource extends Resource
+{
+    protected static ?string $model = GameSet::class;
+
+    protected static ?string $modelLabel = 'Hub';
+    protected static ?string $pluralModelLabel = 'Hubs';
+    protected static ?string $breadcrumb = 'Hubs';
+    protected static ?string $navigationIcon = 'heroicon-m-link';
```

Comment:
I wonder if `fa-sitemap` might be a better icon for hubs.

Reply:
Happy to change the icon to anything - unfortunately `fa-sitemap` seems to throw an error:

>Svg by name "fa-sitemap" from set "default" not found.

<img width="761" alt="Screenshot 2024-12-19 at 9 08 17 PM" src="https://github.com/user-attachments/assets/b8df298f-2367-43d3-af39-01b7da240a66" />

Reply:
`fas-sitemap` worked. I like that a lot better than what I had. Addressed in latest.